### PR TITLE
Feature/integrate mwss transformer

### PR DESCRIPTION
**Pull request by tundish**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,5 @@
+arrow==0.8.0
 cryptography>=1.5.3
```
Comment (JamesGardiner) [inline requirements.txt]: This version of cryptography is out of sync with other services. I was under the impression that we want to keep this the same across services?=.

File: sdx/common/async_consumer.py [hunk 2]
```diff
@@ -0,0 +1,365 @@
+import pika
+import logging
+from structlog import wrap_logger
+from app import settings
+import time
+
+logger = wrap_logger(logging.getLogger(__name__))
```
Comment (JamesGardiner) [inline sdx/common/async_consumer.py]: Define a formater for the logger first - `logging.basicConfig(level=LOGGING_LEVEL,
format=LOGGING_FORMAT)`

Comment (JamesGardiner) [reply sdx/common/async_consumer.py]: This will also initialise and add a streamhandler to stderr

File: sdx/common/async_consumer.py [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+    def on_channel_open(self, channel):
+        """This method is invoked by pika when the channel has been opened.
+        The channel object is passed in so we can make use of it.
+
+        Since the channel is now open, we'll declare the exchange to use.
+
+        :param pika.channel.Channel channel: The channel object
+
+        """
+        logger.info('Channel opened')
```
Comment (JamesGardiner) [inline sdx/common/async_consumer.py]: Can the channel be logged?

File: sdx/common/formats/cs_formatter.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    Creates a receipt in IDBR format.
+
+    """
+
+    form_ids = {
+        "134": "0004",
+    }
+
+    @staticmethod
+    def pck_name(survey_id, seq_nr, **kwargs):
```
Comment (JamesGardiner) [inline sdx/common/formats/cs_formatter.py:14]: Any reason for extra kwargs?

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
A couple of questions here for you. One thing - can we add more error logging on exceptions where they aren't used for flow control?

File: sdx/common/formats/cs_formatter.py (hunk 4 above)
Comment (tundish) [reply sdx/common/formats/cs_formatter.py:14]: It's so you can call it like this:

```
ids = Survey.identifiers(reply)
name = CSFormatter.pck_name(**ids._asdict())
```

Comment (JamesGardiner) [reply sdx/common/formats/cs_formatter.py:14]: 👍

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**